sections/03_categorize.md

```diff
@@ -142,21 +142,34 @@ This indicates a potential strength of deep methods. It may be possible to
 repurpose features from task to task, improving overall predictions as the field
 tackles new challenges.
 
-Several authors have created reusable feature sets for medical terminologies using
-neural embeddings, as popularized by word2Vec [@tag:Word2Vec]. This approach
-was first used on free text medical notes by De Vine et al.
-[@doi:10.1145/2661829.2661974] with results at or better than traditional methods.
-Y. Choi et al.[@doi:10.1145/2567948.2577348] built embeddings of standardized
-terminologies, such as ICD and NDC, used in widely available administrative
-claims data. By learning terminologies for different entities in the same
-vector space, they can potentially find relationships between different
-domains (e.g. drugs and the diseases they treat). Medical claims data does not
-have the natural document structure of clinical notes, and this issue was
-addressed by E. Choi et al. [@doi:10.1145/2939672.2939823], who built
-embeddings using a multi-layer network architecture which mimics the structure
-of claims data. While promising, difficulties in evaluating the quality of
-these kinds of features and variations in clinical coding practices remain as
-challenges to using them in practice.
+Several authors have created reusable feature sets for medical terminologies
+using natural language processing (NLP) and neural embedding models, as
+popularized by Word2vec [@tag:Word2Vec]. This approach was first used on free
+text medical notes by De Vine et al. [@doi:10.1145/2661829.2661974]
+with results at or better than traditional methods. Y. Choi et al.
+[@doi:10.1145/2567948.2577348] built embeddings of standardized terminologies,
+such as ICD and NDC, used in widely available administrative claims data. By
+learning terminologies for different entities in the same vector space, they can
+potentially find relationships between different domains (e.g. drugs and the
+diseases they treat). Medical claims data does not have the natural document
+structure of clinical notes, and this issue was addressed by E. Choi et al.
+[@doi:10.1145/2939672.2939823], who built embeddings using a multi-layer network
+architecture which mimics the structure of claims data. While promising,
+difficulties in evaluating the quality of these kinds of features and variations
+in clinical coding practices remain as challenges to using them in practice.
+
+NLP has also been applied directly to EHRs to predict disease comorbidities and
+phenotypes as well as novel gene - disease associations. Gligorijevic et al
+[@doi:10.1038/srep32404] built neural embedding models and mined more than 35
+million records collected over a decade. The learned disease representations
+significantly ouperformed state-of-the-art methods, achieving close to 86%
```

Comment:
@brettbj Thanks for reviewing the paper and feedback.

My understanding is that the authors try to address two tasks:

1.  Disease2Disease (D2D, or disease phenotyping), where the model is trained exclusively using raw EHR data and learn the disease representations using a method similar to word2vec where each record is a sentence and each disease is a word:
    > In this section we propose the disease2diseases (D2D) approach for learning disease representations, building upon ideas introduced by the recently proposed word2vec algorithm[...] the language model learns representations of diseases in a low-dimensional space using each patient dis- charge record as a “sentence” and the diseases within the record as “words”.

    To _validate_ their model they then use shared GWAS genes to asses how close two diseases really are (but the GWAS data is _not_ used during training): 
    > To train the D2D model, we used only EHR data (P dataset) [...] Specifically, we selected 2,739 diseases found in the GWAS data and for each retrieved K nearest diseases, with ∈K {1,2,5, 10}. Each of the retrieved diseases was labeled as positive if it shares a gene with the query disease, and labeled negative otherwise, which is used as a proxy for having the same phenotype54. Then, we computed precision@K for each disease as a fraction of positive neighbors within the K retrieved ones, and report the average precision over all 2,739 diseases...

2. DiseaseAndGenes2Diseases (DAG2D, or retrieval of disease - gene associations), a model using GWAS data to learn representations of diseases and genes:
    > The DAG2D model assumes that a subset of diseases from the training data D are associated with genes, where
the associations are provided by domain experts and considered as domain knowledge. We leverage this informa- tion by assigning a vector representation to each gene, and make use of disease contexts in the training data to jointly learn both disease vectors and gene vectors in the same low-dimensional space.

    This model is simply validated on a held-out test set:
    > To train the DAG2D model (for both of the tasks), we extended patient discharge records by associ- ating diseases to corresponding genes according to GWAS data [...] We randomly selected 20% of the diseases that have associated genetic data (the diseases found in the GWAS data set) and removed all their gene associations from the training data. Although we removed genetic information, these diseases are not removed from the original EHR data, so that we are able to learn their vector representations. Second, we learn DAG2D on the data set where the remaining 80% of diseases remained associated with their corresponding gene information data. The DAG2D model then contains low-dimensional representations of diseases and genes in the same embedding space, and we evaluated model performance by measuring how many of the removed genetic associations were correctly retrieved

So, given the performance measures and comparisons they provide I think the results of both models are quite impressive: in D2D they recapitulate genetics-based distances between diseases only using EHR data and in DAG2D they convincingly recover known disease genetic associations from data held out during training.

I'd be happy to try and improve my summary if it's unclear: any specific suggestion would be welcome/helpful.

@agapow Sure, I can remove the accuracy figure if that's uninformative. Do we have a paper-wide policy on this?

CC @cgreene @agitter